src/main/scripts/src/app/shared/perspectives/documents/post-it/attribute-tree/attribute-input-element.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * -----------------------------------------------------------------------\
+ * Lumeer
+ *
+ * Copyright (C) since 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+
+export interface AttributeInputElement {
+
+  id: string;
+  x: number;
+  y: number;
```

Comment:
Inside each document the `Input` elements are structured inside table. The x and y refer to the column and row inside it. It's useful for navigation inside the table using keyboard.

Renamed them to `inputTableRow` and `inputTableColumn` (Not using `inputTableX` and `inputTableY` because it can be misleading to some people)